start.sh

```diff
@@ -36,7 +36,14 @@ chown -R backup:backup /data
 # CRONTAB is set in the Dockerfile
 # CRON_SCHEDULE and the other environment variables are set in docker-compose.yml
 echo "" > $CRONTAB
-echo "${BACKUP_SCHEDULE} DB_HOST=${DB_HOST} DB_NAME=${DB_NAME} DB_USER=${DB_USER} DB_PASS=${DB_PASS} KEEP_DAYS=${KEEP_DAYS} /app/backup.sh" >> $CRONTAB
+
+if [ "$BACKUP_CRON_ENABLE" = true ]; then
```

Comment:
suggest using a return early pattern

https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
do you mean like this, or write a more complex solution with actual function returns? https://github.com/MaRDI4NFDI/docker-backup/pull/17/commits/346ead4344301d8a245584965dbfb296cf4b2cb9

Reply:
Yes. The style is correct however now the functionality is broken;-)